concent_api/verifier/tasks.py

```diff
@@ -40,7 +48,7 @@ def blender_verification_order(
     result_size: int,
     result_package_hash: str,
     output_format: str,
-    scene_file: str,  # pylint: disable=unused-argument
+    scene_file: str,
```

Comment:
this whole task has grown dramatically. There's a lot of logic and for me it is very hard to follow what's going on. I would really instist on having it refactored in a way that small functions are extracted and the task consits of calls to such small functions.

Reply:
@dybi Lets discuss this tomorrow.

Reply:
as discussed it will be done in separate issue

Reply:
Issue https://github.com/golemfactory/concent/issues/524